packages/react-native/ReactCommon/react/nativemodule/samples/ReactCommon-Samples.podspec

```diff
@@ -62,7 +61,7 @@ Pod::Spec.new do |s|
   s.source_files = "ReactCommon/**/*.{cpp,h}",
         "platform/ios/**/*.{mm,cpp,h}"
 
-  s.dependency "RCT-Folly"
+  s.dependency "RCT-Folly", folly_version
```

Comment:
this should not be specified. RN should decide the version of Folly.

Reply:
Fixed